File: PHP/Cap05/c05ex03.php
<?php
// c05ex03.php

$I = 1;
while (!($I > 5)) {
  printf("Entre um valor numerico: ");
  fflush(STDOUT);
  $N = intval(trim(fgets(STDIN)));

  $R = $N * 3;

  printf("Resultado = %d\n", $R);
  printf("\n");

  $I++;
}

printf("\n");
printf("Tecle <Enter> para encerrar... ");
trim(fgets(STDIN));

?>

File: RUBY/Cap05/c05ex03.rb
# c05ex03.rb

#!/usr/bin/env ruby

i = 1
until (i > 5)
  print "Entre um valor numerico: "
  n = gets.chomp.to_i

  r = n * 3

  puts "Resultado = #{r}"
  puts

  i += 1
end

puts
print "Tecle <Enter> para encerrar... "
gets
